frontend/src/utils/constants.js

```diff
@@ -13,8 +13,8 @@ export const AuthRoutes = Object.freeze({
 
 export const ApiRoutes = Object.freeze({
   USER: baseRoot + '/user',
-  USER_PERMISSIONS_FACILITIES: baseRoot + '/user/permissions/facilities',
   USER_FACILITIES: baseRoot + '/user/:contactId/facilities',
+  USER_PERMISSIONS_FACILITIES: baseRoot + '/user/permissions/facilities',
```

Comment:
We haven't been too consistent here but it would make sense to reuse the USER constant rather than repeating baseRoot + '/user'.

Reply:
Fixed.